reedsolomon_test.go

```diff
@@ -826,16 +826,34 @@ func testReconstructData(t *testing.T, o ...Option) {
 		t.Fatal(err)
 	}
 
-	// Reconstruct 3 shards with 3 data and 5 parity shards
+	// Reconstruct parity shards from data
 	shardsCopy := make([][]byte, 13)
+	for i := 0; i < 8; i++ {
+		shardsCopy[i] = shards[i]
+	}
+
+	shardsRequired := make([]bool, 13)
+	shardsRequired[10] = true
+
+	err = r.ReconstructSome(shardsCopy, shardsRequired)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !bytes.Equal(shardsCopy[10], shards[10]) {
+		t.Fatal("ReconstructSome did not reconstruct required shards correctly")
+	}
+
+	// Reconstruct 3 shards with 3 data and 5 parity shards
```

Comment:
[question] not directly related to this PR but why does this claim to reconstruct 3 shards when below only two shares are requested for reconstruction:

```go
	shardsRequired = make([]bool, 8)
	shardsRequired[3] = true
	shardsRequired[4] = true
```